apax/data/input_pipeline.py

```diff
@@ -488,61 +478,86 @@ def __init__(
 
         forces = "forces" in label_keys
         stress = "stress" in label_keys
-        self.prepare_batch = BatchProcessor(cutoff, forces, stress)
+        self.prepare_batch = BatchProcessor(
+            cutoff, forces, stress, atom_padding, nl_padding
+        )
 
         self.count = 0
-        self.reset_every = reset_every
+
         self.max_count = self.n_epochs * self.steps_per_epoch()
-        self.buffer = deque()
+        from queue import Queue
```

Comment:
Do we want/need the import here or can it be moved to the module level?

Reply:
nope, I'll move it up